astropy/modeling/tests/test_rotations.py

```diff
@@ -127,7 +128,63 @@ def test_euler_angles(axes_order):
                                  [(s1*s2), (c1*s3 + c2*c3*s1), (c1*c3 - c2*s1*s3)]])
 
                 }
-    model = models.EulerAngleRotation(23.4, 12.2, 34, axes_order)
-    mat = model._create_matrix(phi, theta, psi, axes_order)
+    mat = rotations._create_matrix([phi, theta, psi], axes_order)
 
     assert_allclose(mat.T, matrices[axes_order])  # get_rotation_matrix(axes_order))
+
```

Comment:
Given that the following tests are very JWST specific, perhaps a little context for what v2, v3 means for someone in the future who needs to change tests that isn't familiar with it?

Reply:
done